.github/workflows/dockerbuild.yml

```
@@ -0,0 +1,62 @@
+name: Docker Build and Publish
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          tags: |
+            type=semver,pattern={{version}}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push nginx image
+        uses: docker/build-push-action@v4
+        with:
+          file: ./docker/Dockerfile
+          push: true
+          build-args: |
+            NGINX=1
+            MAPPROXY_VERSION==${{ steps.meta.outputs.tags }}
+          tags: |
+            ghcr.io/mapproxy/mapproxy-nginx:${{ steps.meta.outputs.tags }}
+
+      - name: Build and push devserver image
+        uses: docker/build-push-action@v4
+        with:
+          file: ./docker/Dockerfile
+          push: true
+          build-args: |
+            DEVSERVER=1
+            MAPPROXY_VERSION==${{ steps.meta.outputs.tags }}
```

[untested] to my understanding the target must be specified instead of the env var `DEVSERVER`
```suggestion
          build-args: |
            MAPPROXY_VERSION==${{ steps.meta.outputs.tags }}
          target: development
```